src/routes/distributions.py

```diff
@@ -93,5 +93,5 @@ def list_disease_state_distribution():
     return UJSONResponse(
         DistributionMessage.disease_states_distribution_found,
```

Comment:
Rename this name, could be:
```suggestion
        DistributionMessage.distribution_found,
```